test/unit/tech/html5.test.js

```diff
@@ -925,6 +930,19 @@ QUnit.test('When Android Chrome reports Infinity duration with currentTime 0, re
   tech.el_ = oldEl;
 });
 
+QUnit.test('No error is thrown when `load` is called `immediately` after play', function(assert) {
+  const videoTag = TestHelpers.makeTag();
+  const fixture = document.getElementById('qunit-fixture');
+
+  fixture.appendChild(videoTag);
+  const playerInstance = videojs(videoTag.id, { techOrder: ['html5'] });
+
+  playerInstance.play();
+  // `load` is called as part of `disposeMediaElement`
+  Html5.disposeMediaElement();
```

Comment:
is the promise error enough to cause this test to fail?

Reply:
Updated the test in the latest commit. It indeed fails every time without my fix.